azure_functions_worker/utils/wrappers.py

```diff
@@ -29,16 +34,23 @@ def call(*args, **kwargs):
             if flag_default and not is_envvar_false(flag):
                 return default
             return func(*args, **kwargs)
+
         return call
+
     return decorate
 
 
-def attach_message_to_exception(expt_type: Exception, message: str) -> Callable:
+def attach_message_to_exception(expt_type: Exception, message: str,
+                                debug_logs=None) -> Callable:
     def decorate(func):
         def call(*args, **kwargs):
             try:
                 return func(*args, **kwargs)
             except expt_type as e:
+                if debug_logs is not None:
+                    logger.error(debug_logs)
```

Comment:
good way to add the errors. :) But we won't get the actual message in this context. Right? The exception will be sent to the host and we'll only

Reply:
Updated the debug logs to have the method name